compiler/vm.nim

```diff
@@ -1674,7 +1686,6 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
         bStrVal = bNode[0].sym.name.s.cstring
       else:
         discard
-      # set result
```

Comment:
Ah, so that's a useless comment, got it.

Reply:
That is correct. I don't know why you need to point that out though.